src/map/battle.cpp

```diff
@@ -8747,6 +8748,12 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 							skillratio += 50 * skill_lv;
 						RE_LVL_DMOD(100);
 						break;
+					case CD_DIVINUS_FLOS:
+						skillratio += -100 + 4000 * skill_lv;
+						skillratio += 70 * pc_checkskill(sd,CD_FIDUS_ANIMUS);
+						skillratio += 5 * sstatus->spl;
```

Comment:
Can you confirm the ratio? otherwise :
```suggestion
						skillratio += 5 * sstatus->spl;	// !TODO: unknown ratio
```